en/middleware/xrce_dds.md

```diff
@@ -241,23 +244,90 @@ The topics are release specific (support is compiled into [microdds-client](../m
 While most releases should support a very similar set of messages, to be certain you would need to check the yaml file for your particular release.
 <!-- Jublish the set we use?: https://github.com/PX4/px4_msgs/issues/22 -->
 
-Note that ROS 2/DDS needs to have the _same_ message definitions that were used to create the XRCE-DDS client module in the PX4 Firmware in order to interpret the messages:
+Note that ROS 2/DDS needs to have the _same_ message definitions that were used to create the XRCE-DDS client module in the PX4 Firmware in order to interpret the messages.
+The definition are stored in the ROS 2 interface package [PX4/px4_msgs](https://github.com/PX4/px4_msgs) and they are automatically synchronized by CI on the `main` and release branches.
```

Comment:
@beniaminopozzan It used to be that they included all messages - do they now just contain the ones that are available?

Reply:
@hamishwillee it still includes all messages as it generally simplify the setup. Why? Because if
all messages are included then a user who just wants to add a new topic to the bridge only needs to update dds_topics.yaml without even recompiling the ROS 2 workspace.

But this is open to discussion. For example a good reason to limit the messages copied into px4_msgs only to those in dds_topics.yaml is that there are *a lot* of messages, and the compilation time can be quite annoying. Even more if you are compiling on a Raspberry Pi.

Reply:
Fair enough. I think it worth clarifying that just because a message is in that folder doesn't mean it is built.

```suggestion
All PX4 message definitions are stored in the ROS 2 interface package [PX4/px4_msgs](https://github.com/PX4/px4_msgs), and they are automatically synchronized by CI on the `main` and release branches.
```